lib/active_record_shards/tasks.rb

```diff
@@ -30,13 +40,22 @@
 
   desc "Create the database defined in config/database.yml for the current RAILS_ENV including shards"
   task create: :load_config do
-    ActiveRecord::Base.configurations.to_h.each do |key, conf|
+    configurations = begin
+      case "#{ActiveRecord::VERSION::MAJOR}.#{ActiveRecord::VERSION::MINOR}"
+      when '6.1'
+        ActiveRecord::Base.configurations.configurations.map { |configuration| [configuration.env_name, configuration.configuration_hash] }
+      else
+        ActiveRecord::Base.configurations.to_h
```

Comment:
I think this will fail dramatically with Rails 7?

Reply:
Yes, but ARS doesn't support Rails 7 yet.